.github/workflows/linux_flatpak.yaml

```diff
@@ -0,0 +1,27 @@
+on:
```

Comment:
you don't need the github actions here.

Reply:
it is/was the easiest to test - both the building and updating.. if you don't mind i'd leave them in if I need to do some testing or building it manually.. they are configured to only build on specific branches.